project/src/main/java/com/google/sps/data/VolunteeringOpportunity.java

```diff
@@ -0,0 +1,55 @@
+package com.google.sps.data;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public final class VolunteeringOpportunity {
+  private String name;
+  private Set<String> skillsRequired;
+  private int numSpotsLeft;
+  private Set<User> volunteers;
+
+  public VolunteeringOpportunity(
+      String name, Set<String> requiredSkills, int numSpotsLeft, Set<User> volunteers) {
+    this.name = name;
+    this.skillsRequired = skillsRequired;
+    this.numSpotsLeft = numSpotsLeft;
+    this.volunteers = new HashSet<User>();
```

Comment:
I think you can move the initialization of volunteers to line.10.

Reply:
Done.